tr/login.json

```diff
@@ -1,46 +1,46 @@
 {
-  "privacy": "Privacy policy",
-  "terms": "Terms of service",
-  "meta": " Kiron Open Higher Education gGmbH",
-  "pleaseWait": "Please wait...",
+  "privacy": "Gizlilik Politikası",
+  "terms": "Kullanım Şartları",
+  "meta": "Kiron Open Higher Education gGmbH",
+  "pleaseWait": "Lütfen bekleyin...",
   "initial": {
-    "welcome": "Welcome back",
-    "googleButton": "Continue with Google",
-    "or": "or",
-    "labelEmail": "Email",
-    "helperTextEmail": "You can use 'Continue with Google' to log in",
-    "labelPassword": "Your password",
-    "errorEmail": "Must be a valid email address",
-    "errorKironEmail": "Please use \"Continue with Google\" to log in",
-    "buttonLogin": "Log in",
-    "forgotPassword": "Forgot your password?",
-    "createAccountTitle": "No account yet?",
-    "createAccountLink": "Register",
-    "passwordReset": "You will receive an email in your personal inbox with a new password for your Kiron Google Account soon.",
-    "sessionTerminated": "You're logged out.",
-    "wrongGoogleAccount": "You used the wrong Google account for accessing campus. Please make sure to choose your \"@students.kiron.ngo\" account in the next step!",
-    "logout": "You're logged out.",
-    "loginFailed": "Login failed, please try again."
+    "welcome": "Tekrar hoş geldiniz",
+    "googleButton": "Google ile Devam edin",
+    "or": "ya da",
+    "labelEmail": "Eposta",
+    "helperTextEmail": "Giriş yapmak için ''Continue with Google'' seçeneğini kullanabilirsin",
+    "labelPassword": "Parolan",
+    "errorEmail": "Geçerli bir e-posta adresi olmalı",
+    "errorKironEmail": "Giriş yapmak için lütfen \"Continue with Google\" kullan",
+    "buttonLogin": "Oturum aç",
+    "forgotPassword": "Parolanı mı Unuttun?",
+    "createAccountTitle": "Henüz hesabın yok mu?",
+    "createAccountLink": "Kaydol",
+    "passwordReset": "Kısa bir süre içinde Kiron Google Hesabın için yeni bir şifre içeren bir e-posta alacaksın",
+    "sessionTerminated": "Çıkış yaptın.",
+    "wrongGoogleAccount": "Kampüse erişmek için yanlış Google hesabını kullandın. Lütfen bir sonraki adımda \ "@ students.kiron.ngo \" hesabını seçtiğinden emin ol!",
```

Comment:
@leaichikawa -  \ "@ students.kiron.ngo \"

There should be no space between the \ and the "